packages/manager/modules/pci/src/projects/project/training/jobs/jobs.controller.js

```diff
@@ -1,11 +1,15 @@
 export default class PciTrainingJobController {
   /* @ngInject */
-  constructor(CucCloudMessage, CucRegionService) {
+  constructor(CucCloudMessage, CucRegionService, PciProjectTrainingJobService) {
     this.CucCloudMessage = CucCloudMessage;
     this.CucRegionService = CucRegionService;
+    this.PciProjectTrainingJobService = PciProjectTrainingJobService;
   }
 
   $onInit() {
+    this.PciProjectTrainingJobService.getAll(this.projectId).then((jobs) => {
```

Comment:
We can add this into `jobs.routing.js` resolve section

Reply:
Nop, we want to refresh the job list every times we came back on the page. Behavior not handle if we put it on jobs.controller.js